net-p2p/amule/amule-2.3.2-r4.ebuild

```diff
@@ -99,11 +96,36 @@ src_install() {
 	default
 
 	if use daemon; then
-		newconfd "${FILESDIR}"/amuled.confd amuled
+		newconfd "${FILESDIR}"/amuled.confd-r1 amuled
 		newinitd "${FILESDIR}"/amuled.initd amuled
 	fi
 	if use remote; then
-		newconfd "${FILESDIR}"/amuleweb.confd amuleweb
+		newconfd "${FILESDIR}"/amuleweb.confd-r1 amuleweb
 		newinitd "${FILESDIR}"/amuleweb.initd amuleweb
 	fi
+
+	keepdir /var/lib/${PN}
+	fowners amule:amule /var/lib/${PN}
```

Comment:
You depend on acct-user/amule conditionally, yet call this unconditionally. It results in broken ebuild, 
```
chown: invalid user: ‘amule:amule’
 * ERROR: net-p2p/amule-2.3.2-r4::testworld failed (install phase):
 *   fowners failed
 * 
 * If you need support, post the output of `emerge --info '=net-p2p/amule-2.3.2-r4::testworld'`,
 * the complete build log and the output of `emerge -pqv '=net-p2p/amule-2.3.2-r4::testworld'`.
 * The complete build log is located at '/tmp/portage/net-p2p/amule-2.3.2-r4/temp/build.log'.
 * The ebuild environment file is located at '/tmp/portage/net-p2p/amule-2.3.2-r4/temp/environment'.
 * Working directory: '/tmp/portage/net-p2p/amule-2.3.2-r4/work/aMule-2.3.2'
 * S: '/tmp/portage/net-p2p/amule-2.3.2-r4/work/aMule-2.3.2'

>>> Failed to emerge net-p2p/amule-2.3.2-r4
```